templates/calendar_signup.mustache

```diff
@@ -0,0 +1,111 @@
+{{!
+    This file is part of Moodle - https://moodle.org/
+
+    Moodle is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Moodle is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+}}
+{{!
+    @template mod_observation/calendar_signup
+
+    Example context (json):
+    {
+	"title": "Timeslots",
+	"month": 10,
+	"year": 2021,
+	"monthname": "October",
+	"nextbtn": "<a href='somelink'> Next </a>",
+	"prevbtn": "<a href='somelink'> Prev </a>",
+	"entries": [
+		[{
+			"daynum": 1,
+			"events": [{
+				"start_time_formatted": "Tuesday 10 October 2021 5pm",
```

Comment:
Im not sure these need to be so descriptive, as long as the surrounding info is complete enough. The position, day and number in the cell, and the month header should be enough to give the context so that only the time eg. 5:00-5:30 is needed. Means less scrolling for overflow too